Integrations/integration-HybridAnalysis.yml

```diff
@@ -354,230 +431,631 @@ script:
+                unknownCounter++;
+            else if (key.status == 'malicious')
+                maliciousCounter++;
+        });
+
+        var res = {
+            'scannersContext' : scannersContext,
+            'unknownCounter' : unknownCounter,
+            'maliciousCounter' : maliciousCounter
+        }
+
+        return res;
+    }
+
+
+    //create table entry from response after 'quick-scan-url-results' command
+    function quickScanResultToEntry(response, minMaliciousScanners) {
+
+        var scannersContext = createScannersContext (response)['scannersContext'];
+        var unknownCounter = createScannersContext (response)['unknownCounter'];
+        var maliciousCounter = createScannersContext (response)['maliciousCounter'];
+
+        var whitelist = {
+            'ID': response.whitelist[0].id,
+            'Value' : response.whitelist[0].value
+        }
+
+        var dbotScore = calculateDbotScore(unknownCounter, maliciousCounter, minMaliciousScanners, response.scanners.lenght);
+
+        var context = createQuickScanContext (response, scannersContext, dbotScore, whitelist);
+        var hr = createQuickScanHumanReadable (response, scannersContext, whitelist);
+
+        return {
+            Type: entryTypes.note,
+            ContentsFormat: formats.json,
+            Contents: response,
+            ReadableContentsFormat: formats.markdown,
+            HumanReadable: hr,
+            EntryContext: context
+        };
+    }
+
+    function createContext (response, hybrid_context) {
+        var url = hybrid_context.Url;
+        if (response.submission_type == 'page_url') {
+            var context = {
+                'HybridAnalysis.URL(val.id && val.id == obj.id)': hybrid_context,
+                'URL(val.Data && val.Data == obj.Data)':{
+                    'Data': url
+                }
+            }
+        }
+
+        else {
+            var filename = url.replace(/^.*[\\\/]/, '');
+            var context = {
+                'HybridAnalysis.URL(val.id && val.id == obj.id)': hybrid_context,
+                'URL(val.Data && val.Data == obj.Data)':{
+                    'Data': url
+                },
+                'File(val.id && val.id == obj.id)' : {
+                    'SHA256': response.sha256,
+                    'Name': filename,
+                }
+            }
+        }
+
+        return context;
+    }
+
+
+    //create table entry from response after 'quick-scan-url/file' command
+    function quickScanToEntry(response, url) {
+        var id = response.id;
+        var hybrid_context = {
+            'Url': url,
+            'ID': response.id,
+            'SHA256': response.sha256,
+            'Finished': response.finished,
+            'SubmissionType' : response.submission_type
+        };
+
+        var context = createContext(response, hybrid_context)
+
+        // adding { ID : url } to the integration context for future use
+        var add_to_context = {};
```

Comment:
What is the meaning of this variable? it should be something like newIntegrationContext, so the name of the variable reflects it's role.
Moreover, this is Python naming. Should be addToContext